setup.py

```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere-aws',
-                 version='0.0.0',
+                 version='0.0.1',
```

Comment:
probably need to [raise the version](https://pypi.org/project/cohere-aws/0.0.0/) higher, looks like it is not ready, maybe resume from latest sagemaker version

Reply:
good call, will resume